### PR TITLE
Fix panic when @event is not instance of Event

### PR DESCRIPTION
**Pull request by thesephist**
... in application shell template.

Sometimes @event is set to be an Airtable record, not the ActiveRecord model.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -29,7 +29,7 @@
               <%= inline_icon 'moon-fill', size: home_action_size, style: 'display: none;' %>
             </action>
           <% end %>
-          <% if @event and not @event.partner_logo_url.blank? %>
+          <% if @event and not @event.partner_logo_url&.blank? %>
```
Comment (lachlanjc) [inline app/views/layouts/application.html.erb]: When possible, prefer to omit not & use a method like present?

**Comment by tmb** [comment]
why is `@event` ever an Airtable record? seems like we should never have this case

**Comment by thesephist** [comment]
@tmb yeah that breaks an assumption we have. I didn't merge this because I thought it was a quick fix but ended up being not. We should review the airtable apply page and actually fix issues in it before merging.

**Comment by tmb** [comment]
fixed in #267

**Pull request closed without merging**